client/packages/coldchain/src/Equipment/ListView/CreateAssetModal.tsx

```diff
@@ -175,8 +176,8 @@ export const CreateAssetModal = ({
 
   // when the pagination changes, fetch the next page
   useEffect(() => {
-    fetchNextPage({ pageParam: pagination });
-  }, [fetchNextPage, pagination]);
+    fetchNextPage({ pageParam: pagination.page });
+  }, [fetchNextPage, pagination.page]);
```

Comment:
Changing search params was creating a new pagination object, resulting in multiple pages of the same data 😭

Reply:
Great find!